sdk/storage/azure-storage-blob/azure/storage/blob/_shared/request_handlers.py

```diff
@@ -4,12 +4,12 @@
 # license information.
 # --------------------------------------------------------------------------
 
-from typing import Dict, Optional
-
 import logging
-from os import fstat
+import re
```

Comment:
Not super important but just thought to fix the import ordering while I was here.